test/Microsoft.AspNet.FileProviders.Combined.Tests/TestUtility/MockChangeToken.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Microsoft.Extensions.Primitives;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNet.FileProviders.Combined.Tests.TestUtility
+{
+    internal class MockChangeToken : IChangeToken
```

Comment:
Could you rename this to `TestChangeToken`? `MockChangeToken` sounds like it's using `Moq`